droidlet/dashboard/web/src/index.js

```diff
@@ -21,11 +21,19 @@ import stateManager from "./StateManager";
 import ObjectFixup from "./components/ObjectFixup";
 import MemoryDetail from "./components/Memory/MemoryDetail";
 import Timeline from "./components/Timeline/Timeline";
+import { isMobile } from "react-device-detect";
 
 import "./index.css";
 
 window.React = React;
 window.ReactDOM = ReactDOM;
+
+if (isMobile) {
+  let url = window.location;
+  url += "mobile.html";
```

Comment:
Can we make this `/mobile` in a future PR?

Reply:
It is possible, but I noticed when I did that I got some [ip:address]:[port]//mobile.html (since I think window.location thought there was a / at the end). Could add some conditional logic for the /, if there is a / don't add it, if there is add it